tests/charts/helm_template_generator.py

```diff
@@ -34,7 +34,7 @@
 
 CHART_DIR = str((Path(__file__).parent / ".." / ".." / "chart").resolve())
 
-DEFAULT_KUBERNETES_VERSION = "1.24.7"
+DEFAULT_KUBERNETES_VERSION = "1.23.13"
```

Comment:
Why change this?

Reply:
As @jedcunningham explained - to keep it consistent with the approach we have in Python/DBs (lowest version is default).